tests/transforms/test_tape_expand.py

```diff
@@ -67,11 +67,9 @@ def test_device_and_stopping_expansion(self, mocker):
             qml.U1(0.2, wires=0)
             qml.Rot(*qml.numpy.array([0.5, 0.2, -0.1], requires_grad=True), wires=0)
 
-        spy_device = mocker.spy(dev, "supports_operation")
```

Comment:
No longer uses `supports_operation` to determine whether or not something is supported.